packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -446,7 +446,7 @@ export default {
         },
 
         isAltColour () {
-            return this.headerBackgroundTheme === 'highlight' || (this.headerBackgroundTheme === 'transparent' && !this.navIsOpen);
+            return this.headerBackgroundTheme === 'highlight' || this.headerBackgroundTheme === 'transparent';
```

Comment:
Worth testing this on mobile and tablet as I think this was in there for a use-case on one of those devices. I think it should apply the alt colour on mobile when the nav is opened.

Reply:
Yes, as Ash said it tied up to mobile and tablet device styles. `&& this.isBelowMid` prop can be added instead of removing `&& !this.navIsOpen`